test/box-luatest/ghs_5_exclude__collation_from_public_access_test.lua

```diff
@@ -0,0 +1,22 @@
+local server = require('test.luatest_helpers.server')
+local t = require('luatest')
+local g = t.group()
+
+g.before_all = function(cg)
+    cg.server = server:new({alias = 'master'})
+    cg.server:start()
+end
+
+g.after_all = function(cg)
+    cg.server:stop()
+end
+
+g.test_no__collation_in_public_access = function(cg)
+    cg.server:exec(function()
+        local t = require('luatest')
+
+        -- 2 is 'public' role id
+        local r = box.space._priv:select{2, 'space', box.space._collation.id}
+        t.assert_equals(#r, 0)
```

Comment:
Please also add a test case that checks that the `_collation` space is invisible via net.box schema (`conn.space._collation` is nil).